index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Notes App</title>
+    <link rel="stylesheet" href="style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet"
+      href="https://unicons.iconscout.com/release/v4.0.0/css/line.css">
```

Comment:
Which library you are using ?

Reply:
Here i use unicons icon liberary for adding icon in web page like Add icon delete icon